.github/workflows/release-utils.yml

```
@@ -138,16 +151,49 @@ jobs:
+        if: ${{ success() }}
+        uses: ./.github/actions/report-result
+        with:
+          command: ${{ needs.authorize.outputs.command }}
+          message: "✅ Release notes for version ${{ needs.authorize.outputs.version }} have been synced."
 
+      - name: Report Failure
+        if: ${{ failure() }}
+        uses: ./.github/actions/report-result
+        with:
+          command: ${{ needs.authorize.outputs.command }}
+          message: |
```

actionlint (1.7.12)

[error] property "sync" is not defined in object type {}

(expression)